src/moonlink_connectors/src/pg_replicate.rs

```diff
@@ -254,18 +255,16 @@ impl PostgresConnection {
                 .get_table_copy_stream(&schema.table_name, &schema.column_schemas)
                 .await
                 .expect("failed to get table copy stream");
-            let res = copy_table_stream(
+            copy_table_stream(
                 schema.clone(),
                 stream,
                 &event_sender,
                 start_lsn.into(),
+                table_base_path,
                 None,
             )
-            .await;
-
-            if let Err(e) = res {
-                error!(error = ?e, table_id = src_table_id, "failed to copy table");
-            }
+            .await
+            .expect("failed to copy table");
```

Comment:
The advantage for `expect` over `unwrap` is it allows more context, I would suggest include table name (or table base path) here.